SPECS/less/less.spec

```diff
@@ -10,6 +10,7 @@ URL:            https://www.greenwoodsoftware.com/less
 Source0:        https://www.greenwoodsoftware.com/less/%{name}-%{version}.tar.gz
 Patch0:         CVE-2022-46663.patch
 Patch1:         CVE-2022-48624.patch
+Patch2:         CVE-2024-32487.patch
```

Comment:
Change looks good but this looks like a high severity CVE, so we want to push it through fast-track.

Reply:
But feel free to merge into `main` first - just keep in mind that there's a chance more work may need to be done for a `fasttrack/2.0` -> `main` cherry-pick depending on the differences between the two branches. Happens rarely.

Reply:
i was told fasttrack is closed this week which i why i switched it from fasttrack over to main

Reply:
I think it's only closed until the 2.0 release is published. It hope it won't be a whole week but Jon would know better. Once that's done the critical and high CVEs, which didn't make it with the release, need to be published through fast-track, though.